### PR TITLE
Grafana routing rule

### PR DESCRIPTION
**Pull request by lluunn**
This will expose grafana dashboard at
https://KFDEPLOY.endpoints.PROJECT.cloud.goog/istio/grafana/

/cc @jlewi 
/cc @kunmingg

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2025)

### CODE REVIEW

File: istio/grafana-virtual-service.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: grafana-vs
+  namespace: kubeflow
```
Comment (kunmingg) [inline istio/grafana-virtual-service.yaml]: Is VirtualService namespace scoped or cluster-wise?

Comment (lluunn) [reply istio/grafana-virtual-service.yaml]: namespaced

Comment (kunmingg) [reply istio/grafana-virtual-service.yaml]: So should we make namespace a parameter or ```kubeflow``` should be good for most cases?

Comment (lluunn) [reply istio/grafana-virtual-service.yaml]: I think kubeflow is good for now.

**Pull request merged**